docs/contributor-guide/developer-guide/technical-contribution-guide.md

```diff
@@ -147,25 +147,30 @@ If you're editing files that others have merged changes to you may need to resol
+- Use [conventional comments](https://conventionalcomments.org) and note `(blocking)` vs. `(non-blocking)` comments. There are browser extensions to make using conventional comments easier.
+- Resolve blocking comments after re-reviewing. Minor, non-blocking comments should be resolved by the implementer.
+
+#### Getting reviews on a pull request
 
 It's easy for the intention of code review comments to be unclear or get misinterpreted. To help with communication, reviewers are encouraged to use [conventional comments](https://conventionalcomments.org) and explicitly indicate that comments are `(blocking)`, where the discussion must be resolved for PR to be merged, or `(non-blocking)` where resolving the discussion is optional for the implementer.
 
 #### Adding Reviewers
 
-You can select the reviewer you feel most has context on your PR. If you want a round robin review, add `civiform/developers` as a reviewer and a team member will be chosen. More details on [how to add roundrobin reviews here](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team). Once that is done you can add the name of the team (civiform/developers) in the reviewer box and it will auto assign someone.
-
-#### Responding to comments
-
-When a review has been provided, reply to comments and then re-request a review by clicking the "arrows pointing in a circle" icon next to the reviewers name in the Reviewers section on the Conversation tab.
-
-#### Approval and merging
-
-Reviewers should grant approval if they do not feel additional review is necessary before merging. This does not necessarily mean no more changes are required before merging, but that any further changes are expected to be minor enough to not require review.
-
-If the pull request does not require additional changes, the reviewer should merge it immediately after approving. Otherwise, once they have addressed all comments marked `(blocking)` or `nit`, the pull request author should either merge if able or re-request review and merging from a maintainer if not. Authors are encouraged to at least reply to `(non-blocking)` and `(if-minor)` comments if they do not address them with code changes.
+You can select the reviewer you feel most has context on your PR. If you want a round robin review, add `civiform/developers` as a reviewer and a team member will be chosen. More details on [how to add roundrobin reviews here](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team). Once that is done you can add the name of the team (`civiform/developers`) in the reviewer box and it will auto assign someone. Alternatively, you may use one of the subsystem groups to get a domain expert to review. These are `civiform/auth-system`, `civiform/core-application`, `civiform/deployment-system`, `civiform/developer-tools`, and `civiform/export-api`. There may be others as time goes on.
```

Comment:
very nit, but can we format the domain experts as a list so it's easier to scan?

And should we link to where the domain experts are defined so people know how to add themselves if they want?